README.md

```diff
@@ -110,8 +110,13 @@ httpd = require('http.server').new(host, port[, { options } ])
   type `text/html`, `text/plain` and `application/json`.
 * `display_errors` - return application errors and backtraces to the client
   (like PHP).
-* `log_errors` - log application errors using `log.error()`.
-* `log_requests` - log incoming requests.
+* `log_requests` - log incoming requests. This parameter can receive:
+    - custom log function using for all routes in current instance of http server. The function should receive a format string 
```

Comment:
Faster to read version:

function value, supporting C-style formatting: `log_requests(fmt, ...)`, where `fmt` is a format string and `...` is Lua Varargs, holding arguments to be replaced in fmt.